notifications/github_users_encrypted.csv

```diff
@@ -0,0 +1 @@
gtRauA5OfCuKB7CrHlBANxVDk4szyqYLfSJMA34-SaXphtwRazs2BzZiz5OMWW1ahvwiAwYEHOIsmDZqu0lPBkYdK9_sje6ONNQN5fPoAuH0Ng5SmctX_7vzvUD6zZ9Ky9_5AZFSUx-bcCccjnnTrn9Wht24DlFrIm2QdkCnQdPhsFoxtU5h7Ct8L3UJhc6msz5g1wHOnUW1V7cHJ_Joo6IUkPbFJuTw5ZaW1Vy64urtuBBTCVdpd8ojPCvos8zvMoBYsRjKQtWJFv6W9FXcr8r10lqR0ErWWBcdyrg88XW3oaF7djg07HFWBaZjG4x5kWDqXfoFtn_4h10N0VGLFKaD8TOh1lA7kVUFBr_T5M5qKTn_iVmJuoZcDYgSXusmETCr_dXw24DlZ3wFdg27X3nkM7AzUSbJvtvmHmS7kwJH_3_JWf05zWRwk5H_kTR7rm6n7LRqsThmvH67bc6bK1or44oO3gyx5zTantkJNF6gWCzuZgTSVDP4X00Y7El-QbM6edFa9ilN_3cScotz-NNUU_TKmsqc9lePt7hSZl4YaAsYvkoLpDq8lV3G6LRorAKNDRtlCPbFRUSsHDXN5slI0kRO3_yujNRzNfSUs9Jyao0TQCJQzTLB0uHC4c9GBe1p4HHj3W5hZOajdw-cn1W9a5jpYynD0aqPVHXA0vlP_3nx6GJETn5bV26RZGUkIeSefPW8oOHaFYi3Y83ubG7QS-YuUCh8uV8cG3ELlStRctbGzgc2AJ3S4KZiQb_48MAG4v_o8lcZLgT1uQ-JLfT07jHOyEqZ7yORU4hv8KW2DU_ezPVg7fxzMhkXEcFXW-BaAgGBNFo091O6WL7thykXLkp41p8HebWrVGZYe_EdcioheKvKKlX0sa_zKVenOAnVgu5ZlW1sotRqJfD9Sazpy1rsCTFClkqTwM20afCugRC0wpGY4j5LGKBeptJy7j9Jb4csi8KPfkrLWngXP-8Hz_VjCjqCzWz_O1t8YpxlQaXAekMQqnhV8s5fyNscv5b7dDWikj3tuV3vK1b2OewniypwjGeDQh30uyWIKmvfXzJnM4WWKhYzRRYhZQVFegP_n8OAJZGNuf5_Ic_kYuIdVJym3AOMignjg4O_JiEA9s314_dCvu-b0R6mpeOS_UXDRJ6OXDTV3lslS2W-XCSsXNojGWMAsELNndE1u5R6vyLobLDmjyTW8KzaysPxk-XhDaFfiFzNnJmkNFRT2gYDMmr6_j0fq4Q7JsryNzPjBQcOXoIKuedSJUStkalpzkcSoGKNvrlLe0zEDwwbd0mtNbbtRhIdEYA5BMj9f765wF2SG1BTtOI5ZbdbiGY3er59ZhXoruB7W3yHIa0FpkiIIgLzXgJLNrgsh4Yu48jylajLbJV5DGSsGBr2bFy-uh4vuIhCFpl2ISUzqUw5qdLvr-ueSqGK2_hY1eU9yQK2S4vE4QSbgxG0lDr6DGfER7fO0C8hO6vV1R-E9C1X_PqEhtZAitz0oufDWekmw-GS3yG9YmSZLfJ0kQNEmCNFCKS_K8hK_fQxVjyEdUs7Oj698e7bXNoMus4zsIOTICrYFM97-iDVF5ut6lU88CVdAlUjygK9SN-3IHRsTvjR3SpyjhR6TB-gIrHqmuljC0er-hX5Bvp59jUs63bnCHogpHKEOhKIQfGZNL1FeW9FBuJY85oJtXeGDdSg4-FgeezfxVH8rfx7__gvGV9ycL_fYhFZIlCXtQcxYLE01Gl9UKoKV3_JFDLpUPX_cMqKsl_TqxYh7oIZM2AZmlfTaXyF3-P1FyYYy5REf58oOekbmjCJnEC6Nsy9VuKoYukR0jUGF6QB9vPBBjt8ci01CUiF21QajNnqfB10sk3HMr19oAvZ3lwV3URrmZ3P20OrD_fT0Ska-8PYfUK1JXMmh3VopkAsI5SyFnr293JprhFDy8hXpdWBZcuYHo4J3J5SLkYI2g92ejJQw88ghgQIZCNsvFhABI8YNaeJ-K_fvYrxsNc63Dk0aIiEL1IHINKmUcIGhtIYO-eXoN83dRwxD8K26W-oMNUoqrZqV0YCwMDyO9M6tf3kBQQPrleJaaJqwJnqPLkDsNP2SWbp5JffYLR3WSfCpvu_Rist8Q5cuv9TPTxJSMPBIPRNdZxiwf1eWVCBp65sldf4cuVgtacq1jd9h4QzcGejQTbPsVAqGbShG3bYLyCzZ8wfLvaCnHKX9xxDfaTpPwgEZc9zuuxaYdiAsNEZ1CCjfzhOgMSbo9wQADe64S5g80sJQlcNatziLQX-SfYP2sEUNpp8rnixM5DfEgdkim4DGIoxzYcMYYjuUrywiSL1FL4iPBZxvkvPE6zvAMNC0u9S3ryYS58FCcsAwzRaJ_WEsgbCYARZcRe8Dt_VABj-ZgHxrjt75H6FfMtt2wHZlZiWTxB6L0ZjNMCE5rHORCIeQDKzYtneHGQfHjSxU7BPokh5J4YZ9uHCuFCiKgTfk2zRTGlIfPSAVmZEZfy18EgaDxsZ58qXLfseqGUYQweDgFsLn27wVID550rOCg-R9qPbA8sgO8wA9Hfp9tsbDOqvgLpVNsFkssN1VoRzr0YQx9aplg-roYjuRos8FcRqqNOa3-383TrKDkxWIxAIN9KxNH6hB-SSbVw5chl4jUtKr4As05tCHegBEirbQ88bwQkgqrVurjdM8and4bAr8TNYk83tLSMIHN_G-mnaqhe3B0zwjJjhJkiajr966lM6YV06GLGtWIycLNLazWNviU2-qtRNi_j9VIArZA7PV1m6hGJkvwD_Hrven2_WYkeY_4veuGN84K5KTjd1uV0jiTcGJCWnQmuFi0reYOlvU8DEmicPe4QUkgVsHTxM5Ao_tqiO_VwQeLbs8VI_9tt1xBm9zVjKQSNdKlMos98AwAkss8Xtj0-__LpLr22UKw22bAnIllffnC8zqnRjm-q5lim2gExlXfx8gaYTZUN16NrzaMAO7CecHv4qwF5VAos3HnnyxbU=
```

Comment:
Does this contain information only about the mongodb related people
Or it has info about all the code owners of ballerina-platform repos?

If it has all the data, we can move this to a common place and maintain. (maybe ballerina-release repo)

Reply:
It contains user id data which is common to all the repos. We can move both the Python script and csv file to a common place. I'll add the fix.